RadixWallet/Clients/FactorSourcesClient/FactorSourcesClient+Interface.swift

```diff
@@ -265,14 +265,21 @@ extension FactorSourcesClient {
 		onMnemonicExistsStrategy: ImportMnemonic.State.PersistStrategy.OnMnemonicExistsStrategy,
 		saveIntoProfile: Bool
 	) async throws -> DeviceFactorSource {
+		@Dependency(\.secureStorageClient) var secureStorageClient
+
+		let deviceInfo = secureStorageClient.loadDeviceInfoOrFallback()
 		let factorSource = switch onDeviceMnemonicKind {
 		case let .babylon(isMain):
 			DeviceFactorSource.babylon(
 				mnemonicWithPassphrase: mnemonicWithPassphrase,
-				isMain: isMain
+				isMain: isMain,
+				deviceInfo: deviceInfo
```

Comment:
Was this not changed to be named to `hostInfo`.

I think a merge main into Sargon, and update iOS host code is needed? Would make review easier :)